Calculator/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Yetti on 2023/05/30.
+
+struct CalculatorItemQueue: DoubleStackQueue, CalculateItem {
+    var enQueueStack: [String] = []
+    var deQueueStack: [String] = []
+    
+    var isEmpty: Bool {
+        return enQueueStack.isEmpty && deQueueStack.isEmpty
+    }
+    
+    mutating func enQueue(_ element: String) {
+        enQueueStack.append(element)
+    }
+    
+    mutating func deQueue() -> String? {
+        deQueueStack = enQueueStack.reversed()
+        let deQueueElement = deQueueStack.popLast()
+        enQueueStack = deQueueStack.reversed()
+        return deQueueElement
+    }
+
+    func peek() -> String? {
+        return enQueueStack == [] ? nil : enQueueStack[0]
```

Comment:
out of range 런타임 에러를 방지하기 위해 조건문을 이용하셨군요! 👍 일어날 수 있는 에러를 방지하는 로직 좋습니다! 
사바사겠지만, 저는 다음과 같이 쓰는게 좀 더 읽기 편해보이네요! 어떠신가요? 😁
```suggestion
        return enQueueStack.isEmpty ? nil : enQueueStack[0]
```

Reply:
원비 말씀대로  isEmpty를 활용하는 방향이 가독성이 좋은것 같아 수정하였습니다! 다만 peek메서드는 로직을 바꾸며 전면수정하였고 enQueueStack과 deQueueStack에 요소가 있는가없는가를 판별하기위해 아래처럼 표현해주었습니다.
```swift
deQueueStack.count >= 1
```